websites/V/Vrc.tl/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "937757295453044806",
+		"name": "atsukoro"
+	},
+	"service": "vrc.tl",
+	"description": {
+		"en": "VRChat party instance list for people that want to have overview on upcoming events."
+	},
+	"url": "vrc.tl",
+	"version": "1.0.0",
+	"logo": "https://files.catbox.moe/jxdt00.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) must be a 512x512px square